nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/JCAMPData.java

```diff
@@ -1007,11 +589,12 @@ public void resetAcqOrder() {
 
     @Override
     public String[] getAcqOrder() {
+        //XXX I have no idea what this tries to do.
```

Comment:
Unclear what it is, but has an impact on MultiVecCounter, vector groups, and ultimately vector indexes, which currenly causes me problem when processing 2D files.